geode-core/src/test/java/org/apache/geode/internal/cache/backup/BackupDistributedTest.java

```diff
@@ -113,7 +113,7 @@
   public void setUp() throws Exception {
     vm0 = getHost(0).getVM(0);
     vm1 = getHost(0).getVM(1);
-    vm2 = getHost(0).getVM(2);
+    vm2 = getHost(0).getVM(-1);
```

Comment:
getVM(-1) is a hack that causes the invocations to be executed in the JUnit JVM. You probably don't want to keep this change.

Reply:
Thanks for catching that! Change pushed.